LICENSE

```diff
@@ -0,0 +1,146 @@
+SHOTGUN PIPELINE TOOLKIT SOURCE CODE LICENSE
```

Comment:
Don't we have anything more recent with Autodesk?

Reply:
i don't think we do!